src/components/useList/__stories__/useList.mdx

```diff
@@ -339,22 +378,27 @@ Use it even if the functionality of the `useList` hook seems redundant to you
 -   `id` - required prop. Set `[data-list-item="${id}"]` data attribute. By this it core list engine finds elements to scroll to.
 -   `title` - base required prop to use. If passed string, applas default component styles according desig system. Pass you own componnet if you wont custom behaviour;
 -   `as` - if needed, override `html` tag. By default - `li`;
--   `size` - the size of the element. By default, `m`. Available options are `s`, `m`, `L`, `xl`. It also affects the radii of the fillets;
+-   `size` - the size of the element. This also affects the rounding radius of the list element . By default, `m`.
+    ##### Size available options:
```

Comment:
That's incorrect to put heading inside the list. Just use regular sentence: `Available sizes are "s", "m", "l"`